pkg/entity/datapolicy/datapolicy.go

```diff
@@ -149,6 +149,40 @@ func evaluate(cmd *cobra.Command) error {
 	return common.Print(printer, err, response)
 }
 
+func transpile(cmd *cobra.Command) error {
+	dataPolicyId, err := cmd.Flags().GetString(common.DataPolicyIdFlag)
+	platformId, err := cmd.Flags().GetString(common.ProcessingPlatformFlag)
+	dataPolicyFileName, err := cmd.Flags().GetString(common.InlineDataPolicyFlag)
+	if err != nil {
+		return err
```

Comment:
Nope, I've removed that in a previous PR, because the error needs to be propagated to the root, as we deal with exit codes there. This is to know the exit code and set it in the telemetry before we actually exit the program.